app/dlc-suredbits-client/src/main/scala/org/bitcoins/sbclient/Exchange.scala

```diff
@@ -0,0 +1,119 @@
+package org.bitcoins.sbclient
```

Comment:
We should probably just open source this stuff in a nice library we don't have to update this internally and externally everytime.

Reply:
problem is internally there is a lot more to an `Exchange` than just what is here, not sure we can reuse